source/includes/wp-api-v3/_order-refunds.md

```diff
@@ -70,13 +72,41 @@ curl -X POST https://example.com/wp-json/wc/v3/orders/723/refunds \
 	-u consumer_key:consumer_secret \
 	-H "Content-Type: application/json" \
 	-d '{
-  "amount": "10"
+  "order_refund": {
```

Comment:
We only create wraps like that in our legacy API, this is intentional here?

Reply:
I think I confused request and response, that `"order_refund"` doesn't belong here, indeed. I've fixed it.